Bundler/Processor/ProcessorService/AssetCreator/App.config

```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <configSections>
+    <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
+      
+      <section name="AssetCreator.UnitySettings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
+      <section name="AssetCreator.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
+    </sectionGroup>
+  </configSections>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+  </startup>
+  <userSettings>
+    
+    <AssetCreator.UnitySettings>
+      <setting name="UnityInstallLocation" serializeAs="String">
+        <value>C:\Program Files\Unity\Editor\</value>
+      </setting>
+      <setting name="UnityBundleProject" serializeAs="String">
+        <value>C:\Bundler\Projects\Unity\</value>
+      </setting>
+      <setting name="UnityBundleProjectCopy" serializeAs="String">
+        <value>C:\Bundler\Projects\Unity\Assets\toBundle\</value>
+      </setting>
+      <setting name="UnityBundleProject_5_6" serializeAs="String">
+        <value>C:\Bundler\Projects\Unity_5_6\</value>
+      </setting>
+      <setting name="UnityBundleProjectCopy_5_6" serializeAs="String">
+        <value>C:\Bundler\Projects\Unity_5_6\Assets\toBundle\</value>
+      </setting>
+      <setting name="UnityInstallLocation_5_6" serializeAs="String">
+        <value>C:\Program Files\Unity_5_6_0f1\Editor\</value>
+      </setting>
+    </AssetCreator.UnitySettings>
+    <AssetCreator.Settings>
+      <setting name="MaxTargetPlatforms" serializeAs="String">
+        <value>VR-PC,HoloLens</value>
+      </setting>
+      <setting name="MaxTargetVertexCount" serializeAs="String">
+        <value>1000000,100000</value>
+      </setting>
+      <setting name="MaxScriptLocation" serializeAs="String">
+        <value>@\"C:\Bundler\Assets\HoloAssetCreator-OSS.ms\"</value>
+      </setting>
+      <setting name="MaxInstallLocation" serializeAs="String">
+        <value>C:\Program Files\Autodesk\3ds Max 2017\</value>
+      </setting>
+      <setting name="BaseDataPath" serializeAs="String">
+        <value>C:\bundler\data</value>
```

Comment:
Same comment as Unity path

Reply:
Added code to dump these settings on start to make is clear what's going where.